pager/src/main/java/com/google/accompanist/pager/PagerState.kt

```diff
@@ -627,12 +627,11 @@ class PagerState(
          * The default [Saver] implementation for [PagerState].
          */
         val Saver: Saver<PagerState, *> = listSaver(
-            save = { listOf<Any>(it.pageCount, it.currentPage, it.currentPageOffset) },
+            save = { listOf<Any>(it.pageCount, it.currentPage) },
```

Comment:
Is the `Any` type still needed now that these are all `Int`?

Reply:
Fixed.